### PR TITLE
New monster

### PR DESCRIPTION
**Pull request by yanndo**

### CODE REVIEW

File: Doug/Monsters/Hobos/Hobo.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{
+    public class Hobo : Monster
+    {
+        public const string MonsterId = "hobo";
+
+        public Hobo()
+        {
+            Id = MonsterId;
+            Name = "Hobo";
+            Description = "Little vapoting punk. Watch outm he can spit on you";
```
Comment (FlyingJello) [inline Doug/Monsters/Hobos/Hobo.cs]: typo

**Comment by yanndo** [comment]
Faite

File: Doug/Monsters/MonsterFactory.cs [hunk 2]
```diff
@@ -3,6 +3,7 @@
 using System.Linq;
 using Doug.Monsters.Brigands;
 using Doug.Monsters.Seagulls;
+using Doug.Monsters.Hobos;
```
Comment (FlyingJello) [inline Doug/Monsters/MonsterFactory.cs]: put it in brigands folder please

File: Doug/Monsters/Hobos/Hobo.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        public const string MonsterId = "hobo";
+
+        public Hobo()
+        {
+            Id = MonsterId;
+            Name = "Hobo";
+            Description = "Little vapoting punk. Watch out, he can spit on you !!";
+            Image = "https://imgur.com/a/ovnZdsK";
+            Level = 10;
+            ExperienceValue = 250;
```
Comment (FlyingJello) [inline Doug/Monsters/Hobos/Hobo.cs]: lower exp value to 100

File: Doug/Monsters/Hobos/Hobo.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            MaxAttack = 69;
+            Hitrate = 18;
+            Dodge = 18;
+            Defense = 17;
+            Resistance = 9;
+
+            DropTable = new Dictionary<LootItem, double>
+            {
+                { new LootItem(Apple.ItemId, 1), 0.05 },
+                { new LootItem(CoffeeCup.ItemId, 3), 0.05 },
```
Comment (FlyingJello) [inline Doug/Monsters/Hobos/Hobo.cs]: 0.2 for the coffee and apple

File: Doug/Monsters/MonsterFactory.cs (hunk 2 above)
Comment (yanndo) [reply Doug/Monsters/MonsterFactory.cs]: Done

File: Doug/Monsters/Hobos/Hobo.cs (hunk 3 above)
Comment (yanndo) [reply Doug/Monsters/Hobos/Hobo.cs]: Done

**Pull request merged**